### ISSUE TITLE
Is 1.19.2 support?

### ISSUE THREAD

**Issue by Bebeno8485** [JustAlittleWolf/ServerPingerFixer#2]
Modrinth says that the latest version of the mod supports 1.19–1.19.4, but when I try to run the mod on 1.19.2, the mod crushes:

```
 Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'Server Pinger Fixer' (serverpingerfixer) 1.0.0+1.19.4 with any version that is compatible with:
		 - minecraft 1.19.2
Unmet dependency listing:
	 - Mod 'Server Pinger Fixer' (serverpingerfixer) 1.0.0+1.19.4 requires any version between 1.19.4 (inclusive) and 1.20- (exclusive) of 'Minecraft' (minecraft), but only the wrong version is present: 1.19.2!
	at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:51)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:192)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:148)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

```

**Comment by JustAlittleWolf** [comment]
Will fix when I get home

**Comment by JustAlittleWolf** [comment]
Is now fixed! You can get the updated version from modrinth or github releases
Fixed with 6c0e92fb5def32006f7f2e1136ae1067416613c6
